docs/source/examples/broadcast.ipynb

```diff
@@ -0,0 +1,225 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "Broadcast Example\n",
+    "=================\n",
+    "\n",
+    "Let's start off in the usual way and create a RabbitMQ communicator and subscribe to receive broadcasts."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import kiwipy\n",
+    "comm = kiwipy.connect('amqp://127.0.0.1')\n",
+    "\n",
+    "def subscriber(_comm, body, sender, subject, _corr_id):\n",
+    "    print(\"Broadcast received:\")\n",
+    "    print(\"sender:\\t{}\\nsubject:{}\\nbody:\\t{}\\n\".format(sender, subject, body))\n",
+    "\n",
+    "\n",
+    "sub_id = comm.add_broadcast_subscriber(subscriber)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "Now, let's broadcast!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Broadcast received:\n",
+      "sender:\tme\n",
+      "subject:yo momma\n",
+      "body:\tso fat\n",
```

Comment:
Actually, given that we want to publish this, I think it is best to change this example. I don't care personally about the language, but it is not the classiest.

Reply:
Aww, no fun!  Fine - changed to a Futurama reference.